samples/sample_vpp/include/sample_vpp_config.h

```diff
@@ -125,6 +125,15 @@ typedef struct
 
 } sDenoiseParam;
 
+struct sColorFillParam: public mfxExtVPPColorFill {
+    FilterConfig mode;
+    sColorFillParam():mfxExtVPPColorFill(), mode(VPP_FILTER_DISABLED)
+    {
+        Header.BufferId = MFX_EXTBUFF_VPP_COLORFILL;
+        Header.BufferSz = sizeof(mfxExtVPPColorFill);
+    };
```

Comment:
No need in semicolon after function body

Reply:
It's not my idea, it's a style of whole file.

Reply:
No, but do it in the code you modify.

Reply:
done